File: 01-HelloWorld-WindowsForms/01 - HelloWorld-WindowsForms-VisualBasic/Form1.vb
Imports _01___HelloWorld_WindowsForms_Csharp



Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim nombre As String
        nombre = Me.TextBox1.Text
        'MessageBox.Show("Hola " & nombre)

        Dim persona1 As New clsPersona
        persona1.nombre = "Angel"
        persona1.apellido = "David"

        MessageBox.Show("Hola " & persona1.nombre & " " & persona1.apellido)
    End Sub
End Class

File: 01-HelloWorld-WindowsForms/01 - HelloWorld-WindowsForms-Csharp/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace _01___HelloWorld_WindowsForms_Csharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void label1_Click(object sender, EventArgs e)
        {

        }

        private void btnSaludar_Click(object sender, EventArgs e)
        {
            string nombre;

            nombre = this.txtNombre.Text;

            //MessageBox.Show("Hola " + nombre);
            //MessageBox.Show($"Hola {nombre}");
            //MessageBox.Show(string.Format("Hola {0}", nombre));

            clsPersona oPersona = new clsPersona();
            oPersona.nombre = "Angel";
            oPersona.apellido = "Gel";

            MessageBox.Show($"Hola, soy el objeto {oPersona.nombre} {oPersona.apellido}");

        }
    }
}
